### PR TITLE
[DPC-168]  Adds cover images for new BO manual for pagoPA

### PR DESCRIPTION
**Pull request by certevol**
#### Screenshots (if appropriate):
**Mobile**: 
image

**Desktop**:
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .changeset/weak-grapes-scream.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"nextjs-website": patch
```
Comment (kin0992) [inline .changeset/weak-grapes-scream.md:2]:
```suggestion
"nextjs-website": minor
```
you are adding a new guide, so it's similar to a "new feature" then it should be a minor

**Review by kin0992 — CHANGES_REQUESTED** [review]

File: .changeset/weak-grapes-scream.md (hunk 1 above)
Comment (certevol) [reply .changeset/weak-grapes-scream.md]: Thanks. I should have changed it, right?

Comment (kin0992) [reply .changeset/weak-grapes-scream.md]: yep!

**Review by kin0992 — APPROVED** [review]

**Review by marcobottaro — CHANGES_REQUESTED** [review]
The images are in the wrong folder. You should move them into `apps/nextjs-website/public/images` directory:
image

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**